code/jest.config.base.js

```diff
@@ -0,0 +1,113 @@
+const os = require('os');
+const path = require('path');
+
+// TODO Revisit this test later, when we have a windows machine @valentinpalkovic
+const skipOnWindows = [
+  'lib/core-server/src/utils/stories-json.test.ts',
+  'lib/core-server/src/utils/StoryIndexGenerator.test.ts',
+  'lib/cli/src/helpers.test.ts',
+  'lib/core-server/src/utils/__tests__/server-statics.test.ts',
+  'lib/core-common/src/utils/__tests__/template.test.ts',
+  'addons/storyshots/storyshots-core/src/frameworks/configure.test.ts',
+  'lib/core-common/src/utils/__tests__/interpret-files.test.ts',
+  'lib/builder-manager/src/utils/files.test.ts',
+  'lib/cli/src/helpers.test.ts',
+  'lib/core-server/src/utils/__tests__/server-statics.test.ts',
+  'lib/core-common/src/utils/__tests__/template.test.ts',
+  'addons/storyshots/storyshots-core/src/frameworks/configure.test.ts',
+  'lib/core-common/src/utils/__tests__/interpret-files.test.ts',
+  'lib/builder-manager/src/utils/files.test.ts',
```

Comment:
This array has been reduced by @valentinpalkovic in #19777

Reply:
I guess it's a race to see which PR is merged first.  😅

Reply:
GO @valentinpalkovic GO!!!

Reply:
🏎️

Reply:
I went ahead and removed the same lines in this PR as well.